apps/s_server.c

```diff
@@ -1499,9 +1499,11 @@ int s_server_main(int argc, char *argv[])
         case OPT_ALPN:
             alpn_in = opt_arg();
             break;
+#ifndef OPENSSL_NO_SRTP
```

Comment:
"close it up" so that only the body is ifdefd out.  keep the case and break.  that avoids compiler warnings.